### DISCUSSION TITLE
AppService Linux

### DISCUSSION THREAD

**Discussion by carloscds** [shibayan/appservice-acmebot#388] [Q&A] [answered]
Hi,

This service create SSL for AppService running on Linux, but using Office 365 Domain or AWS domain ?

Thanks.

**Comment by shibayan** [comment, marked as answer]
Azure DNS is required for App Service on Linux in this application. If you want to use other DNS providers, please use the Key Vault version.
https://github.com/shibayan/keyvault-acmebot
